src/MetaMorpho.sol

```diff
@@ -386,14 +384,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             totalSupplied += suppliedAssets;
```

Comment:
Maybe we should add :
`if (totalWithdrawn == totalSupplied) break;`

Reply:
It is unexpected to have more markets to supply to than to withdraw from so I'm against